apps/src/music/blockly/MusicBlocklyWorkspace.js

```diff
@@ -324,28 +324,54 @@ export default class MusicBlocklyWorkspace {
   }
 
   updateHighlightedBlocks(playingBlockIds) {
+    if (!this.workspace) {
```

Comment:
Copying over comments from https://github.com/code-dot-org/code-dot-org/pull/56224 for visibility
> @bencodeorg: Any value to differentiating these warning messages?

Oh that's a good idea, I'll add some specifics